src/impl_methods.rs

```diff
@@ -154,8 +154,8 @@ where
 
     /// Return an uniquely owned copy of the array.
     ///
-    /// If the input array is contiguous, then theoutput array will have the same
-    /// memory layout. Otherwise, the layout ofthe output array is standard C.
+    /// If the input array is contiguous, then the output array will have the same
+    /// memory layout. Otherwise, the layout ofthe output array is unspecified.
```

Comment:
```suggestion
    /// memory layout. Otherwise, the layout of the output array is unspecified.
```
And so on, there's a missing space on every line

Reply:
So sorry for that missing

Reply:
No problem. It's a good PR if both the maintainer and contributor have time and patience for reviewing in several cycles. Even though it of course feels awkward to request more changes etc. For an open source project I think on balance it makes sense to require high quality - contributors are in general not known if they will stay around and tie up loose ends and we avoid merging half-baked features. And merging PRs with too many loose ends that the maintainer has to fix - that burns out the maintainer. That's the sum of my experience.

Reply:
Thank you very much, it means a lot to me.